tests/GridFS/StreamWrapperFunctionalTest.php

```diff
@@ -169,6 +169,29 @@ public function testWritableStreamStat()
         $stat = fstat($stream);
         $this->assertSame(6, $stat[7]);
         $this->assertSame(6, $stat['size']);
+  }
+
+    public function testWritableStreamStatAfterSaving()
+    {
+        $id = new ObjectId;
+        $stream = $this->bucket->openUploadStream('filename', ['chunkSizeBytes' => 1024, '_id' => $id]);
+
+        $this->assertSame(6, fwrite($stream, 'foobar'));
+        $this->assertTrue(fclose($stream));
+        $currentTimestamp = time();
+        $stream = $this->bucket->openDownloadStream($id);
```

Comment:
I just realized that the `setUp()` function for this test creates a file with `_id` of `length-10` and `uploadDate` of `new UTCDateTime('1484202200000')`. Switching over to that would reduce all of these lines to a single `$stream = $this->bucket->openDownloadStream('length-10');`, and allow the ObjectID import to be removed as well.

Apologies for not catching this earlier. I hadn't looked at the entire file until now.